externs/shaka/player.js

```diff
@@ -1015,3 +1029,28 @@ shaka.extern.PlayerConfiguration;
  * @exportDoc
  */
 shaka.extern.LanguageRole;
+
+
+/**
+ * @typedef {{
+ *   height: number,
+ *   positionX: number,
+ *   positionY: number,
+ *   uris: !Array.<string>,
+ *   width: number
+ * }}
+ *
+ * @property {number} height
+ *    The thumbnail height in px.
+ * @property {number} positionX
+ *    The thumbnail left position in px.
+ * @property {number} positionY
+ *    The thumbnail top position in px.
+ * @property {!Array.<string>} uris
+ *   An array of URIs to attempt.  They will be tried in the order they are
+ *   given.
+ * @property {number} width
```

Comment:
nit: maybe, group height and width together? This throws me off a bit :)
(As in:
 *   height: number,
 *   width: number,
 *   positionX: number,
 *   positionY: number,
 *   uris: !Array.<string>)

Reply:
Do you prefer an object with dimensions? It can be done, but I do not see a gain compared to what I have right now. 
Can you help me understand why you want this change?